posts/2023/july/linguist/2023-07-13-linguist.md

```diff
@@ -13,58 +13,58 @@ image:
 lang:
 ---
 
-Browser is most used program by me and by most users of internet, so it is important for browser to be high efficient for daily routine. Altough browsers have embedded translation features and exists a lot of browser extensions for translation, all of them have bad quality and absolutely not acceptable for intensive use. This is why i've create Linguist - ultimate browser extension for translation.
+The web-browser is possibly the most used program for me and for most users of the internet, so it is important for browser to work efficiently during daily use. Altough most browsers have embedded translation features and there exists a lot of browser extensions providing translation capabalities, I have found them to, in my opinion, not good enough and unacceptable for intensive use. This is why I have created Linguist - the ultimate browser extension for translation.
```

Comment:
Not "possible" and not "in my opinion",  i declare that it is. The imperative text style in head are important to hook readers. Hook must be a sharp to explain the pain that motivates me to do things and to motivates a people to read the post. Let's fix it.